recon_surf/README.md

```diff
@@ -20,10 +20,10 @@ List them by running the following command:
 ### Required arguments
 * --sd: Output directory \$SUBJECTS_DIR (equivalent to FreeSurfer setup --> $SUBJECTS_DIR/sid/mri; $SUBJECTS_DIR/sid/surf ... will be created).
 * --sid: Subject ID for directory inside \$SUBJECTS_DIR to be created ($SUBJECTS_DIR/sid/...)
-* --t1: T1 full head input (not bias corrected). 
+* --t1: T1 full head input (not bias corrected). This must be conformed (dimensions: 256x256x256, voxel size: 1x1x1, LIA orientation, and data type UCHAR). Images can be conformed using FastSurferCNN's [conform.py](https://github.com/Deep-MI/FastSurfer/blob/master/FastSurferCNN/data_loader/conform.py) script.
```

Comment:
Add command to do this (python3 conform.py --input <unconformed-T1> --outptu <conformed-T1>)